### PR TITLE
Don't help people depend on tidyverse or devtools

### LINKED ISSUE #172: use_package("tidyverse|devtools") should generate a warning
And suggest using the individual core packages instead

### PR DESCRIPTION
**Pull request by jennybc**
Fixes #172 use_package("tidyverse|devtools") should generate a warning

So far, I'm just refusing to add the dependency and not adding all the individual packages. I don't really think we want people depending on every single package in the tidyverse either. I think it's OK to ask people to figure out what they need, yes?

### CODE REVIEW

File: R/package.R [hunk 1]
```diff
@@ -15,6 +15,9 @@
 #' use_package("dplyr", "suggests")
 #' }
 use_package <- function(package, type = "Imports") {
+  refuse_package(package, verboten = "tidyverse")
+  refuse_package(package, verboten = "devtools")
```
Comment (jennybc) [inline R/package.R]: Is it too early to do this for devtools, i.e. before more of the pieces have made it to CRAN?

Comment (hadley) [reply R/package.R]: Yeah, I think it's too early.

**Comment by jennybc** [comment]
@batpigandme Yes, it would be good to have the body of that email as a blog post, so we have a definitive link and explanation for why there should be few / no reverse dependencies on the meta-packages. I won't use it in this PR but will use it elsewhere.

**Comment by batpigandme** [comment]
Cool – draft post that is literally the email and a line introducing it is here: https://github.com/tidyverse/tidyverse.org/pull/161

**Pull request merged**